### DISCUSSION TITLE
how to use this plugin using tailwindcss cdn?

### DISCUSSION THREAD

**Discussion by sea-sol** [tailwindlabs/tailwindcss#6652] [Help]
it is working perfectly. but I need to use this using cdn tailwindcss.
so I have to use tailwindcss config like this
```
<script src="https://cdn.tailwindcss.com"></script>
  <script>
    tailwind.config = {
      theme: {
        extend: {
          colors: {
            clifford: '#da373d',
          }
        }
      }
    }
  </script>
```

but the problem is 
```
plugins: [
            require('@tailwindcss/line-clamp'),
        ],
```
never work with this way, is there any solution?

**Comment by dani-ooo** [comment]
i think this can be achieved by adding it to the URL: `https://cdn.tailwindcss.com?plugins=line-clamp`
